react/features/shared-video/components/native/SharedVideoButton.ts

```diff
@@ -94,25 +87,25 @@ class VideoShareButton extends AbstractButton<Props, *> {
  * @param {Object} state - The Redux state.
  * @param {Object} ownProps - The properties explicitly passed to the component instance.
  * @private
- * @returns {Props}
+ * @returns {IProps}
  */
-function _mapStateToProps(state, ownProps): Object {
+function _mapStateToProps(state: IReduxState, ownProps: any) {
     const { ownerId, status: sharedVideoStatus } = state['features/shared-video'];
-    const localParticipantId = getLocalParticipant(state).id;
+    const localParticipantId = getLocalParticipant(state)?.id;
     const enabled = getFeatureFlag(state, VIDEO_SHARE_BUTTON_ENABLED, true);
     const { visible = enabled } = ownProps;
 
     if (ownerId !== localParticipantId) {
         return {
-            _isDisabled: isSharingStatus(sharedVideoStatus),
+            _isDisabled: isSharingStatus(sharedVideoStatus ?? ''),
```

Comment:
updating `isSharingStatus` to accept optional argument looks like a cleaner approach